qt-bugs@ issue : N199952
Trolltech task ID : 199967
applied: yes
author: Thiago

--- a/src/corelib/io/qurl.cpp
+++ b/src/corelib/io/qurl.cpp
@@ -3805,14 +3805,14 @@
     if (!QURL_HASFLAG(d->stateFlags, QUrlPrivate::Parsed))
         return d->encodedOriginal.isEmpty();
     else
-        return d->scheme.isEmpty()
-        && d->userName.isEmpty()
-        && d->password.isEmpty()
-        && d->host.isEmpty()
+        return d->scheme.isEmpty()   // no encodedScheme
+        && d->userName.isEmpty() && d->encodedUserName.isEmpty()
+        && d->password.isEmpty() && d->encodedPassword.isEmpty()
+        && d->host.isEmpty()   // no encodedHost
         && d->port == -1
-        && d->path.isEmpty()
+        && d->path.isEmpty() && d->encodedPath.isEmpty()
         && d->query.isEmpty()
-        && d->fragment.isEmpty();
+        && d->fragment.isEmpty() && d->encodedFragment.isEmpty();
 }
 
 /*!
