qt-bugs@ issue :
Trolltech task ID : 199635
applied: yes
author: Trolltech

--- a/src/corelib/global/qglobal.h
+++ b/src/corelib/global/qglobal.h
@@ -1419,7 +1419,7 @@
 
 #if !defined(Q_ASSERT)
 #  ifndef QT_NO_DEBUG
-#    define Q_ASSERT(cond) if(!(cond))qt_assert(#cond,__FILE__,__LINE__)
+#    define Q_ASSERT(cond) ((!(cond)) ? qt_assert(#cond,__FILE__,__LINE__) : qt_noop())
 #  else
 #    define Q_ASSERT(cond) qt_noop()
 #  endif
@@ -1433,7 +1433,7 @@
 
 #if !defined(Q_ASSERT_X)
 #  ifndef QT_NO_DEBUG
-#    define Q_ASSERT_X(cond, where, what) if(!(cond))qt_assert_x(where, what,__FILE__,__LINE__)
+#    define Q_ASSERT_X(cond, where, what) ((!(cond)) ? qt_assert_x(where, what,__FILE__,__LINE__) : qt_noop())
 #  else
 #    define Q_ASSERT_X(cond, where, what) qt_noop()
 #  endif
