author: David Faure <faure@kde.org>

Fix compilation with CXXFLAGS=-pedantic -Wno-long-long.

--- a/src/network/kernel/qnetworkproxy.cpp
+++ b/src/network/kernel/qnetworkproxy.cpp
@@ -224,7 +224,7 @@
 #endif
 };
 
-Q_GLOBAL_STATIC(QGlobalNetworkProxy, globalNetworkProxy);
+Q_GLOBAL_STATIC(QGlobalNetworkProxy, globalNetworkProxy)
 
 class QNetworkProxyPrivate
 {
--- a/src/network/socket/qsocks5socketengine.cpp
+++ b/src/network/socket/qsocks5socketengine.cpp
@@ -351,7 +351,7 @@
     QHash<int, QSocks5BindData *> store;
 };
 
-Q_GLOBAL_STATIC(QSocks5BindStore, socks5BindStore);
+Q_GLOBAL_STATIC(QSocks5BindStore, socks5BindStore)
 
 QSocks5BindStore::QSocks5BindStore()
     : mutex(QMutex::Recursive)
--- a/src/corelib/xml/qxmlstream_p.h
+++ b/src/corelib/xml/qxmlstream_p.h
@@ -116,7 +116,7 @@
 
     GOTO_INDEX_OFFSET = 418,
     GOTO_INFO_OFFSET = 925,
-    GOTO_CHECK_OFFSET = 925,
+    GOTO_CHECK_OFFSET = 925
   };
 
   static const char  *const spell [];
