author: David Faure <faure@kde.org>

Fix compilation with CXXFLAGS=-pedantic -Wno-long-long.

--- a/src/network/qnetworkproxy.cpp
+++ b/src/network/qnetworkproxy.cpp
@@ -207,7 +207,7 @@
 #endif
 };
 
-Q_GLOBAL_STATIC(QGlobalNetworkProxy, globalNetworkProxy);
+Q_GLOBAL_STATIC(QGlobalNetworkProxy, globalNetworkProxy)
 
 class QNetworkProxyPrivate
 {
--- a/src/network/qsocks5socketengine.cpp
+++ b/src/network/qsocks5socketengine.cpp
@@ -345,7 +345,7 @@
     QHash<int, QSocks5BindData *> store;
 };
 
-Q_GLOBAL_STATIC(QSocks5BindStore, socks5BindStore);
+Q_GLOBAL_STATIC(QSocks5BindStore, socks5BindStore)
 
 QSocks5BindStore::QSocks5BindStore()
     : mutex(QMutex::Recursive)
--- a/src/xml/qdom.cpp
+++ b/src/xml/qdom.cpp
@@ -676,7 +676,7 @@
 
     *ok = true;
     return result;
-};
+}
 
 // [15] Comment ::= '<!--' ((Char - '-') | ('-' (Char - '-')))* '-->'
 // can't escape "--", since entities are not recognised within comments
--- a/src/xml/qxmlstream_p.h
+++ b/src/xml/qxmlstream_p.h
@@ -116,7 +116,7 @@
 
     GOTO_INDEX_OFFSET = 418,
     GOTO_INFO_OFFSET = 925,
-    GOTO_CHECK_OFFSET = 925,
+    GOTO_CHECK_OFFSET = 925
   };
 
   static const char  *const spell [];
