--- a/src/network/ssl/qsslsocket_openssl_symbols.cpp
+++ b/src/network/ssl/qsslsocket_openssl_symbols.cpp
@@ -296,13 +296,13 @@
         return false;
     }
 #else
-    QLibrary libssl(QLatin1String("ssl"));
+    QLibrary libssl(QLatin1String("ssl.so.0.9.8"));
     if (!libssl.load()) {
         // Cannot find libssl
         return false;
     }
 
-    QLibrary libcrypto(QLatin1String("crypto"));
+    QLibrary libcrypto(QLatin1String("crypto.so.0.9.8"));
     if (!libcrypto.load()) {
 #ifndef QT_NO_DEBUG
         // Cannot find libcrypto
