--- a/src/corelib/global/qlibraryinfo.cpp
+++ b/src/corelib/global/qlibraryinfo.cpp
@@ -79,12 +79,7 @@ class QLibraryInfoPrivate
 {
 public:
     static QSettings *findConfiguration();
-    static void cleanup()
-    {
-        QLibrarySettings *ls = qt_library_settings();
-        if (ls)
-            ls->settings.reset(0);
-    }
+    static void cleanup();
     static QSettings *configuration()
     {
         QLibrarySettings *ls = qt_library_settings();
@@ -92,6 +87,13 @@ public:
     }
 };
 
+void QLibraryInfoPrivate::cleanup()
+{
+    QLibrarySettings *ls = qt_library_settings();
+    if (ls)
+        ls->settings.reset(0);
+}
+
 QLibrarySettings::QLibrarySettings()
     : settings(QLibraryInfoPrivate::findConfiguration())
 {
--- a/src/corelib/tools/qhash.h
+++ b/src/corelib/tools/qhash.h
@@ -520,7 +520,7 @@ Q_INLINE_TEMPLATE void QHash<Key, T>::de
 }
 
 template <class Key, class T>
-Q_INLINE_TEMPLATE void QHash<Key, T>::duplicateNode(QHashData::Node *node, void *newNode)
+void QHash<Key, T>::duplicateNode(QHashData::Node *node, void *newNode)
 {
     Node *concreteNode = concrete(node);
     if (QTypeInfo<T>::isDummy) {
