Author: Pino Toscano <pino@kde.org>
Description: Identify PowerPCSPE as PowerPC.
Bug-Debian: http://bugs.debian.org/cgi-bin/bugreport.cgi?bug=623185
Forwarded: no
Origin: vendor
Last-Update: 2011-04-27

--- a/configure
+++ b/configure
@@ -3013,7 +3013,7 @@
             CFG_HOST_ARCH=x86_64
         fi
         ;;
-    *:*:ppc)
+    *:*:ppc|*:*:powerpcspe)
         if [ "$OPT_VERBOSE" = "yes" ]; then
             echo "    32-bit PowerPC (powerpc)"
         fi
